packages/text-area/theme/lumo/vaadin-text-area-styles.js

```diff
@@ -56,8 +56,20 @@ const textArea = css`
     --_lumo-text-field-overflow-mask-image: none;
   }
 
-  /* Vertically align icon prefix/suffix with the first line of text */
-  [part='input-field'] ::slotted(vaadin-icon) {
+  /* Use sticky positioning to keep prefix/suffix/clear button visible when scrolling textarea container */
+  [part='input-field'] ::slotted([slot='prefix']),
+  [part='input-field'] ::slotted([slot='suffix']),
```

Comment:
nit: could simplify this as follows (we use this e.g. in `vaadin-input-container` styles).

```suggestion
  [part='input-field'] ::slotted([slot$='fix']),
```

Reply:
Done